examples/demo/src/themes/ThemeSwipper.tsx

```diff
@@ -0,0 +1,51 @@
+import {
+    FormControl,
+    InputLabel,
+    MenuItem,
+    Select,
+    SelectChangeEvent,
+    Stack,
+} from '@mui/material';
+import { useContext } from 'react';
+import { ToggleThemeButton, useStore, useTranslate } from 'react-admin';
+import { ThemeContext, ThemeType, themes } from './themeContext';
+
+export const ThemeSwipper = () => {
```

Comment:
```suggestion
export const ThemeSwapper = () => {
```

Don't forget to rename the file